### ISSUE TITLE
[ADE Support] 508 Audit Report for 43731 Critical Defect 5

### ISSUE THREAD

**Issue by lakper** [department-of-veterans-affairs/va.gov-team#106461]
> **QUICK SUMMARY:** _Ticket created to track feedback on 508 audit findings for ticket #106395 _

## 508 Audit Feedback Received

> - [Ref. Section 508 ICT Testing Baseline Portfolio All Baseline Tests for Web.](https://ictbaseline.access-board.gov/allwebbaselines.html)
> - [Ref. Impact Table](https://www.section508.gov/content/mapping-wcag-to-fpc/)

Project Name: Yellow Ribbon Program information in Comparison Tool VASI #: 2103
Audit Number: 43731 Audit Date: 3/31/2025
Audit Scope: General
Product Type: Web Applications
Project Status: Open
Project Site: https://www.va.gov/education/gi-bill-comparison-tool/institution/31937132
Project Version:
Release Recommendation: No
Release Recommendation Comments: The defects are listed in the following report.
Link to 508 Compliance Application: [508 Compliance Application](https://apps.gov.powerapps.us/play/e/default-e95f1b23-abaf-45ee-821d-b7ab251ab3bf/a/5f7da63e-384d-4ce4-8cc7-f3be6c60c363?tenantId=e95f1b23-abaf-45ee-821d-b7ab251ab3bf&sourcetime=2023-08-14%2017%3A50%3A55Z&RecordID=2673)

**Defect 5 | Critical | Created:**
_WCAG 4.1.2 Name, Role, Value | For all user interface components (including but not limited to: form elements, links and components generated by scripts), the name and role can be programmatically determined; states, properties, and values that can be set by the user can be programmatically set; and notification of changes to these items is available to user agents, including assistive technologies._

On the COLUMBIA UNIVERSITY IN THE CITY OF NEW YORK: GI Bill® Comparison Tool | Veterans Affairs screen, the names for some elements cannot be understood by assistive technology—some elements have the same name. For example, there are two buttons with the name Learn more about how we calculate the number of GI Bill students—the second instance opens the Accreditation and why it matters dialog. (New: 03/31/2025)

On the COLUMBIA UNIVERSITY IN THE CITY OF NEW YORK: GI Bill® Comparison Tool | Veterans Affairs screen, the names for some elements cannot be understood by assistive technology. For example, when tab is used to navigate to the first set of radio buttons, “group No radio button checked 2 of 2” is read—the question is not read. (New: 03/31/2025)

On the COLUMBIA UNIVERSITY IN THE CITY OF NEW YORK: GI Bill® Comparison Tool | Veterans Affairs screen, the names for some elements cannot be understood by assistive technology. For example, when tab is used to move the focus from Search Accredited Attorneys, Claims Agents, or Veterans Service Organizations (VSO) Representatives, “(opens in a new tab) opens in a new tab Link” is read. (New: 03/31/2025)

### Ticket Updates
- [x] _Connect to an `Epic/Parent` or `Intake` (what body of work is this a part of?) If this is the `Epic/Parent`, connect related tickets as sub-issues._
- [ ] _If this is the `Epic/Parent` ticket, add label for `ADE 508 RECHECK`_
- [x] _If this ticket is a `Child` ticket aka `sub-issue` please add the appropriate labels for `ADE 508 FLAG CRITICAL`;`ADE 508 FLAG HIGH`; `ADE 508 FLAG MEDIUM` or `ADE 508 FLAG LOW`_
- [x] _Label with `Originator/Team` (product team or stakeholder requesting support; ADE if internal)_
- [x] _Label date in the `Open Date` field_
- [ ] _Label with `Estimate` (level of effort expected for this ticket)_
- [ ] _Add `Assignee(s)` name(s) to ticket_ (include Lakwi)
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [ ] _Select a `Priority Level`_
- [x] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`
- [ ] _Make sure the `ADE 508 RECHECK` label is on this ticket upon closing and that a follow-up date has been established and calendared._

**Comment by coforma-jamie** [comment]
[Findings in parent ticket comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/106395#issuecomment-2773380490)
